Include/Acidanthera/Library/OcConfigurationLib.h

```diff
@@ -407,11 +406,39 @@ typedef enum {
   OC_ARRAY (OC_MISC_TOOLS_ENTRY, _, __)
   OC_DECLARE (OC_MISC_TOOLS_ARRAY)
 
+///
+/// Reference:
+/// https://github.com/acidanthera/bugtracker/issues/1954#issuecomment-1084220743
```

Comment:
Probably it is better to mention this 41-byte thing in docs, then remove these links in the comment. Same below.